tools/download-srs.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+# Script to download and verify SRS files for EigenDA
+# Usage: ./download-srs.sh <size_in_bytes>
+# Example: ./download-srs.sh 16777216  # for 16MB
+
+set -euo pipefail
+
+# Maximum allowed size (set to 16GB)
+MAX_SIZE_BYTES=$((16 * 1024 * 1024 * 1024))
+
+# Check for required dependencies
+function check_dependencies() {
+    local missing_deps=0
+    for cmd in curl sha256sum awk grep tr; do
+        if ! command -v "$cmd" &> /dev/null; then
+            echo "Error: Required command '$cmd' not found"
+            missing_deps=1
+        fi
+    done
+    
+    if [ $missing_deps -eq 1 ]; then
+        echo "Please install missing dependencies and try again."
```

Comment:
I would error inside the for loop and print the missing dependency instead.

Reply:
No longer relevant, after migration to go cli tool, instead of bash